Comment in the script is revealing: "GNU sed has many extentions not on osX and FreeBSD.
If you change near sed, test on FreeBSD." At least GNU grep is needed, but better use GNU
tools upfront.

--- ntpsec-1.1.3/ntpclients/ntpleapfetch	2019-03-05 07:03:40.171055770 +0000
+++ ntpsec-1.1.3/ntpclients/ntpleapfetch.new	2019-03-05 20:12:35.795799604 +0000
@@ -47,7 +47,7 @@ LOGFAC=daemon
 # Places to look for commands.  Allows for CRON having path to
 # old utilities on embedded systems
 
-PATHLIST="/opt/sbin:/opt/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:"
+PATHLIST="/usr/gnu/bin:/opt/sbin:/opt/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:"
 
 REQUIREDCMDS=" wget logger tr sed"
 
